src/Illuminate/Database/Schema/Grammars/SqlServerGrammar.php

```diff
@@ -733,6 +733,17 @@ public function typeMultiPolygon(Fluent $column)
         return 'geography';
     }
 
+    /**
+     * Create the column definition for a generated virtual column type.
+     *
+     * @param  \Illuminate\Support\Fluent  $column
+     * @return string|null
+     */
+    protected function typeVirtual(Fluent $column)
```

Comment:
The parent class should also have this method and throw a "not supported" exception (see `Builder` or `SQLiteGrammar` for examples).